packages/core/content-manager/admin/src/pages/EditView/components/FormLayout.tsx

```diff
@@ -23,9 +23,11 @@ export const ResponsiveGridItem = styled(Grid.Item)`
   }
 `;
 
-interface FormLayoutProps extends Pick<EditLayout, 'layout'> {}
+interface FormLayoutProps extends Pick<EditLayout, 'layout'> {
+  hasBackground?: boolean;
```

Comment:
i'm open to suggestions for better names. the issue is that the form layout is normally rendered on top of a greyish background (for light mode), so it has borders and shadows to separate each layout pane from it. We don't want those styles for preview though, because the layout is rendered on a pure white card.

Reply:
Maybe a crazy idea but what if instead of a prop we used the path params to determine whether or not it is a the preview page and if it is then adjust the styles

Reply:
I don't think it works because of relations on the fly. We don't want the borders etc on the relation modal, and those modals can be displayed both in the edit view path and the preview path

Reply:
Bad idea, won't work for relations otf.

Reply:
but again, I'm sure there must be a better name for that prop 😁

Reply:
I need the same stuff for the modal, so for me hasBackground can be the best option because it is the need we have in both the features

Reply:
btw in my case I don't have path params because the relation's modal is inside the edit view page